src/components/RecordDetail/MediaContents.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
```

Comment:
Would it make sense to move these components from `components/*` to something like `apps/pages/*`? The idea of the `apps` directory is to isolate components that are specific to certain parts of the app, but not necessarily used anywhere else.